services/QuillLMS/app/services/demo/report_demo_creator.rb

```diff
@@ -269,8 +270,15 @@ def self.create_teacher(email)
     )
   end
 
+  def self.create_user_email_verification(teacher)
+    teacher.verify_email(UserEmailVerification::STAFF_VERIFICATION)
+  end
+
   def self.create_teacher_info(teacher)
-    teacher_info = TeacherInfo.create(user: teacher, minimum_grade_level: 'K', maximum_grade_level: 12)
+    # setting the demo teacher as their own verifier if they don't belong to a school
+    verifier = teacher.school&.schools_admins&.first&.user_id || teacher
```

Comment:
Should the left side of this `||` be a `User` object instead of `user_id` since we call `verifier.id` below?